Source/AIFlatSwitch.swift

```diff
@@ -15,6 +15,11 @@ import UIKit
     
     // MARK: - Public
     
+    /**
+     circleLayer, is the layer which appears inside the circle.
+     */
+    @IBInspectable open var circleLayer = CAShapeLayer()
```

Comment:
Is there any chance you can rename this to something like backgroundLayer and have an @IBInspectable backgroundLayerColor?

Reply:
Like this:

```
@IBInspectable open var backgroundLayer = CAShapeLayer()
    
/**
     Color for the inner circle.
*/
@IBInspectable open var backgroundLayerColor: UIColor = UIColor.clear
```

Reply:
Yes exactly and you need to implement didSet{} of backgroundLayerColor to react to changes on the go